CHANGELOG.md

```diff
@@ -808,7 +820,7 @@ No breaking changes have been introduced in this release.
 ### Summary
 
 Debut release of decK
-
```

Comment:
Please keep this line jump, the format will otherwise gets messed up: https://github.com/Kong/deck/blob/ce763b0c709e52f18a4e3597a950260ce37465ff/CHANGELOG.md#summary-3